.github/workflows/cpp.yml

```diff
@@ -276,7 +276,7 @@ jobs:
       ARROW_GANDIVA: ON
       # google-could-cpp uses _dupenv_s() but it can't be used with msvcrt.
       # We need to use ucrt to use _dupenv_s().
-      # ARROW_GCS: ON
+      ARROW_GCS: OFF
```

Comment:
Could you revert this change because `ARROW_GCS` is `OFF` by default in `ci/scripts/cpp_build.sh`?

Reply:
done.